src/app/pages/waiting-room-to-car/components/vehicle-checks-question/vehicle-checks-question.ts

```diff
@@ -94,4 +122,21 @@ export class VehicleChecksQuestionComponent implements OnChanges {
   shouldShowOutcomeFields(): boolean {
     return !!(this.questionResult && this.questionResult.code && this.questionResult.description);
   }
+
+  isInvalid(): boolean {
+    if (this.submitClicked) {
+      return !this.questionFormControl.valid || !this.questionOutcomeFormControl.valid;
+    }
+    return false;
+  }
+
+  displayErrorMessage(): boolean {
+    return this.submitClicked && this.questionFormControl.valid && !this.questionOutcomeFormControl.valid;
+  }
+
+  setInvalidIfUndefined(control: UntypedFormControl) {
+    if (control.value === undefined) {
+      control.setErrors({ 'invalid': true });
+    }
+  }
```

Comment:
Do we need this? Can we not use standard form control way and mark both controls as `[Validators.required]`?